sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -520,6 +522,20 @@ def process(
             negative_prompts=all_negative_prompts,
         )
 
+        if opts.dp_write_raw_template:
+            # if original_prompt / original_negative_prompt will not be auto quoted by webui and will be effected by stripped, we need to quote it ourselves
+            if original_prompt and original_prompt != original_prompt.strip() and original_prompt == generation_parameters_copypaste.quote(original_prompt):
+                original_prompt = json.dumps(original_prompt, ensure_ascii=False)
+            if original_negative_prompt and original_negative_prompt != original_negative_prompt.strip() and original_negative_prompt == generation_parameters_copypaste.quote(original_negative_prompt):
+                original_negative_prompt = json.dumps(original_negative_prompt, ensure_ascii=False)
+
+            p.extra_generation_params.update(
+                {
+                    "Template": original_prompt if original_prompt else None,
+                    "Negative Template": original_negative_prompt if original_negative_prompt else None,
+                }
+            )
```

Comment:
I think there's no point putting Nones in the dict here:
```suggestion
            if original_prompt:
                p.extra_generation_params["Template"] = original_prompt
            if original_negative_prompt:
                p.extra_generation_params["Negative Template"] = original_negative_prompt
```

Reply:
true